tests/codebuild/run_integtest.sh

```diff
@@ -92,7 +93,7 @@ if [ "${need_setup_cluster}" == "true" ]; then
     readonly cluster_region="us-east-1"
 
     # By default eksctl picks random AZ, which time to time leads to capacity issue.
-    eksctl create cluster "${cluster_name}" --nodes 2 --node-type=c5.4xlarge --timeout=40m --region "${cluster_region}" --zones us-east-1a,us-east-1b,us-east-1c --auto-kubeconfig --version=1.14 --fargate 
+    eksctl create cluster "${cluster_name}" --nodes 1 --node-type=c5.xlarge --timeout=40m --region "${cluster_region}" --zones us-east-1a,us-east-1b,us-east-1c --auto-kubeconfig --version=1.14 --fargate 
```

Comment:
nit: Remove `--nodes` and `--node-type` from this statement - this is a Fargate cluster so they should never have been there in the first place (oops, my bad)

Reply:
Yep, Gautam and I discussed this yesterday but was unsure if it was intentional